assets/chat/js/formatters.js

```diff
@@ -20,21 +20,25 @@ class EmoteFormatter {
 
     format(chat, str, message=null){
         if (!this.regex) {
-            const emoticons = [
-                ...chat.emoticons,
-            ].join('|');
+            const emoticons = Array.from(chat.emoticons).join('|');
             const suffixes = Object.keys(GENERIFY_OPTIONS).join('|');
-            this.regex = new RegExp(`(^|\\s)(${emoticons})(:(${suffixes}))?(?=$|\\s)`, 'gm');
+            this.regex = new RegExp(`(?:^|\\s)(${emoticons})((?::(?:${suffixes}))*)(?=$|\\s)`, 'gm');
         }
-        
-        return str.replace(this.regex, function(m) {
-            if (!m.includes(':')) {
-                const emote = m.replace(/\s/g, '');
+
+        return str.replace(this.regex, match => {
+            const index_of_suffix = match.indexOf(':');
```

Comment:
camelCase like other variables?

Reply:
I don't think camel case is the standard here.